src/python/dxpy/bindings/dxproject.py

```diff
@@ -380,6 +387,8 @@ def update(self, name=None, summary=None, description=None, protected=None,
         :type database_ui_view_only: boolean
         :param external_upload_restricted: If provided, whether project members can upload data to project from external sources, e.g. outside of job
         :type external_upload_restricted: boolean
+        :param database_results_restricted: If provided, minimum amount of data that project members with VIEW access can see from databases in the project
```

Comment:
seems that `unset_database_results_restricted` documentation is missing

Reply:
Fixed it for unset_database_results_restricted and unset_allowed_executables. Thanks!